dist/README.md

```diff
@@ -0,0 +1,41 @@
+# Grist Cloud Distribution
+
+This a comprehensiveversatile distribution of Grist made with Docker Compose. It
+combines the following tools as services in Compose:
+
+* Traefik as a reverse proxy and optionally for TLS termination (handling HTTPS)
+* Authelia as an email + password manager, optionally with 2FA, password resets
+* Dex as an OIDC provider, by default only using Authelia as a backend
+  but optionally can also use Google or Microsoft as a identity
+  provider
+
+# Quickstart
+
+First, make sure that [docker is installed](https://docs.docker.com/engine/install/).
+
+In order to set up authentication secrets and some basic, default configuration, first run
+
+```sh
+./bin/generate-secure-secrets
+```
+
+<!-- TODO: Logically separate out the crypto part of setup from the
+other bits required, such as APP_HOME_URL or persist location-->
+This will use Authelia to generate some cryptographic information
+needed in `config/secrets`, as well as an `.env` file in the current
+directory with some defaults for running the Compose image. You may
+wish to inspect the `.env` file to change some defaults, but leave the
+cryptographic choices alone. Then run as usual,
+
+```sh
+docker compose up
+```
+
+This should start, in this order, Traefik, Authelia, Dex, and finally
+Grist, at the URL specified by `APP_HOME_URL` (by default, at
+`http://grist.localhost`). You should then be able to visit that URL
+in a web browser and log in to Grist.
+
+<!--TODO: let user configure the default username/password -->
+Currently, the default usernmae to log into Authelia is `test` with
```

Comment:
Can you emphasize the defaults a bit more. Once I saw "docker compose up" I went off to try that, and only came back here to try and find a username/password. I then missed the URL in the sentence above and used `localhost`. If you put a block with:

Default URL: http://grist.localhost
Default username: test
Default password: test

there's a better chance of the first run being pleasingly successful.

Reply:
Sure. But I don't expect this README will be much of an end-user experience in the end. I hope it won't be.